### PR TITLE
Create prioritization.md

### PR DESCRIPTION
**Pull request by davelab6**
Publicly document our prioritization rubric

### CODE REVIEW

File: gf-guide/prioritization.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+An existing high usage family has a bug, and the fix will improve pages/documents already using it in a meaningful way?
+
+This will get a "P2" tag, and is given priority to be completed within the planned quarter, given no pending P0 or P1 projects.
+These are expected to make up a small share of activity per quarter.
+
+Target schedule: Quarterley, but can slip.
+
+### P3: General Priority
+
+A nice project that [@davelab6](https://github.com/davelab6), [@chrissimpkins](https://github.com/chrissimpkins) or others in the Google Fonts team would like to ship sooner than later?
```
Comment (chrissimpkins) [inline gf-guide/prioritization.md:59]: IMO just indicate "Google Fonts team" and remove all individual names.  This keeps it evergreen.  You never know... :)

**Review by chrissimpkins — CHANGES_REQUESTED** [review]

File: gf-guide/prioritization.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+## Table of contents
+{:.no_toc}
+* TOC goes here
+{:toc}
+
+## Priority Definitions
+
+### P0: Top Priority
+
+A large Google Product is blocked from achieving some goal, or soon to be blocked?
```
Comment (rsheeter) [inline gf-guide/prioritization.md:24]: General: excessive use of ? rather than .

File: gf-guide/prioritization.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+It depends.
+An issue can move from P3 to P2 to P1 based on the severity of the problem, combined with the popularity in context (eg, popularity in CSS API, or Android Downloadable Fonts, within Workspace editors, etc).
+This is a judgment call when evaluating the issue: If the answer isn't obvious, whoever is assigning priorities should consult with their manager.
+
+### How will we communicate onboarding priorities and known deadlines during our quarterly planning process?
+
+A Google Fonts team [Program Manager](https://www.google.com/search?q=Program+Manager) will directly track, communicate and coordinate as needed, including maintaining an internal spreadsheet of priority P1 and P2 projects that is shared with the wider Google Fonts team at the beginning of each quarter, based on Github Issues tags.
+If the list requires updates during the current quarter, the Program Manager will communicate changes to the Google Fonts internal leadership team within one week of the plan update.
+One of the primary goals of this communication plan is to surface all known deadline driven onboarding work as early as possible to the team.
```
Comment (rsheeter) [inline gf-guide/prioritization.md:94]: I think perhaps this is THE primary goal, we want everyone to be very clear on that

**Review by rsheeter — APPROVED** [review]
LGTM with nits.

Optional: I prefer the table form in the upstream doc, could reproduce it that way here too

File: gf-guide/prioritization.md (hunk 2 above)
Comment (davelab6) [reply gf-guide/prioritization.md]: Fixed in table

File: gf-guide/prioritization.md (hunk 3 above)
Comment (davelab6) [reply gf-guide/prioritization.md]: Fixed

**Comment by davelab6** [comment]
> LGTM with nits.
> 
> Optional: I prefer the table form in the upstream doc, could reproduce it that way here too

Table'ized :)

**Review by chrissimpkins — CHANGES_REQUESTED** [review]

**Comment by vv-monsalve** [comment]
@chrissimpkins, I've committed your suggested fixes. Would you like to unblock the merge here?

**Review by chrissimpkins — APPROVED** [review]

**Comment by chrissimpkins** [comment]
All set.  Thanks Viv.

**Pull request merged**